Sources/Misc/Deprecations.swift

```diff
@@ -56,6 +56,64 @@ public extension Purchases {
         return await eligiblePromotionalOffers(forProduct: product)
     }
 
+    /**
+     * Enable debug logging. Useful for debugging issues with the lovely team @RevenueCat.
+     */
+    @available(*, deprecated, message: "use Purchases.logLevel instead")
+    @objc static var debugLogsEnabled: Bool {
```

Comment:
[not for this PR] I'm surprised the others in this file don't have `@objc`, did we miss that?

Reply:
made it for this PR 🤷 since the original work got undone anyway